src/utils/familyCode.spec.ts

```diff
@@ -0,0 +1,18 @@
+import {
+  convertCodeNumberToFamilyCode,
+  extractCodeNumberFromFamilyCode,
+} from './familyCode';
+
+describe('FamilyCode util', () => {
+  it('should convert code number to family code', () => {
+    expect(convertCodeNumberToFamilyCode(99)).toBe('F00099');
+    expect(convertCodeNumberToFamilyCode(1)).toBe('F00001');
+    expect(convertCodeNumberToFamilyCode(99999)).toBe('F99999');
```

Comment:
What about
```ts
convertCodeNumberToFamilyCode(123456)
convertCodeNumberToFamilyCode(-1)
convertCodeNumberToFamilyCode(NaN)
```

Reply:
first one is out of range, because family code is 6 character string which one of them is F.

Reply:
for the third one, the expected result could be an exception.
but the first one what should we do? exception is sounds good to me.